charts/nodejs/templates/deployment.yaml

```diff
@@ -57,6 +59,9 @@ spec:
         - name: K8S_ENV
           value: {{ .Values.kubernetes.env | quote }}
         
+        - name: SANDBOX_ID
+          value: {{ .Values.biomageCi.sandboxId | quote }}
+
```

Comment:
Make sure sandbox ID is accessible as an env variable to deployments (ui, api) if they need to use it.